rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -70,23 +72,22 @@ def setUp(self):
       "organization": {
         "display": None,
         "reference": "Organization/PITT",
-      },
-      "site": [{
-        "display": None,
-        "reference": "mayo-clinic",
-      }],
-      "identifier": [{
-        "system": "http://any-columbia-mrn-system",
-        "value": "MRN456"
-      }]
+      }
     }
 
+  def test_pairing_summary(self):
+    participant = self.send_post('Participant', {"providerLink": [self.provider_link]})
+    participant_id = participant['participantId']
+    path = 'Participant/%s' % participant_id
+    participant['awardee'] = 'PITT'
+    particpant_update = self.send_put(path, participant, headers={'If-Match': 'W/"1"'})
+    self.assertEquals(particpant_update['awardee'], participant['awardee'])
```

Comment:
This checks the summary for awardee, but isn't checking organization or site; and it's not checking that you can query by organization or site and get back participants.

You can add tests in here:

https://github.com/all-of-us/raw-data-repository/blob/8e897fb23127085dc3ff964ae7e70b75cf7e6ff8/rest-api/test/unit_test/api_test/participant_summary_api_test.py#L471

to do this sort of thing...

Reply:
Please add tests of the ParticipantSummary endpoint, around where I linked to above.